codemon/CodemonHelp.py

```diff
@@ -9,4 +9,6 @@ def showHelp():
   print("codemon init <contestName> - - - - - - initialises a contest")
   print("codemon init -n <file> - - - - - - - - creates file with given name")
   print("codemon listen - - - - - - - - - - - - compiles and gives output")
-  print("codemon practice <dirName> - - - - - - Starts a practice session")
\ No newline at end of file
+  print("codemon practice <dirName> - - - - - - Starts a practice session")
+  print("codemon practice <dirName> - - - - - - Starts a practice session")
+  print("codemon reg - - - - - - - - - - - - -  Creates a file with all your details")
```

Comment:
You've written codemon practice help message twice here. Also codemon reg doesn;t look good in bottom take it up after codemon listen. Give the description as : "Register user details".